### PR TITLE
Docs for new operators

### PR DESCRIPTION
**Pull request by saqimtiaz**
- Docs for `is[draft]` operator
- Docs for `toggle[]` operator
- Docs for `search-replace[]` operator
- Error handling improvement for `search-replace` suffixes
- Tests for `search-replace` operator

### CODE REVIEW

**Comment by saqimtiaz** [comment]
I've consolidated all remaining outstanding docs in this PR, in addition to #4969

**Comment by Jermolene** [comment]
Thanks @saqimtiaz shall I merge this now or is there more to come?

**Comment by saqimtiaz** [comment]
@Jermolene this is everything and both this and #4969 can be merged.

**Comment by Jermolene** [comment]
Thanks @saqimtiaz

File: editions/tw5.com/tiddlers/filters/toggle Operator.tid [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+modified: 20201107154300086
+op-input: a list of items
+op-output: the input list with the title  <<.place R>> toggled
+op-parameter: an item to toggle in the input list
+op-parameter-name: R
+op-purpose: toggle the title specified in the operand in the input
+tags: [[Filter Operators]] [[Listops Operators]] [[Order Operators]]
+title: toggle Operator
+type: text/vnd.tiddlywiki
+
```
Comment (kookma) [inline editions/tw5.com/tiddlers/filters/toggle Operator.tid:12]: @saqimtiaz  I cannot find the position of below typo in https://tiddlywiki.com/prerelease/#toggle%20Operator

image

The `search-replace` shall be changed by `toggle` 

Sorry I could not find the position to make a PR.

Comment (saqimtiaz) [reply editions/tw5.com/tiddlers/filters/toggle Operator.tid:12]: @kookma its in a field. If you can make a PR please do or I will take care of it a little later.

Comment (saqimtiaz) [reply editions/tw5.com/tiddlers/filters/toggle Operator.tid:12]: @kookma thanks #5020

**Pull request merged**